### DISCUSSION TITLE
Best Way to Get `MainWindow.UseStudio()` Outside of App.cs?

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [unoplatform/uno#19098] [Q&A] [answered]
I am attempting to move `MainWindow.UseStudio()` to an external (development/debug-only) assembly and am having trouble getting the reference to make it resolve.  This method lives in `Uno.UI.HotDesign.Client` but it is not available as a NuGet package.  So, I cannot add it as a reference to the target assembly. 🤔

Any assistance/guidance/pointers to get this to resolve would be greatly appreciated. 🙏

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. At this time, this is not supported and this call must be made from the main assembly in the app.xaml.cs.

Why do you need to place it outside?

**Reply by Mike-E-angelo** [reply]
Doh, that's a bummer thank you for the bad news there @jeromelaban I appreciate you taking the time to do so.

In my case, I am not a fan of compiler directives and avoid them if I can.  Instead, I use specific assemblies for different environments and place the configurations there.  Nice and organized + tidy.  I would certainly appreciate any consideration toward allowing this call outside of the primary assembly. 👍🙏
